[2024-01-31 14:58:49,599] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.aggregate_dim_table manual__2024-01-31T14:58:24.526071+00:00 [queued]>
[2024-01-31 14:58:49,606] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.aggregate_dim_table manual__2024-01-31T14:58:24.526071+00:00 [queued]>
[2024-01-31 14:58:49,606] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 14:58:49,607] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-01-31 14:58:49,607] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 14:58:49,616] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): aggregate_dim_table> on 2024-01-31 14:58:24.526071+00:00
[2024-01-31 14:58:49,620] {standard_task_runner.py:52} INFO - Started process 8735 to run task
[2024-01-31 14:58:49,622] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'aggregate_dim_table', 'manual__2024-01-31T14:58:24.526071+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpn_nno7mo', '--error-file', '/tmp/tmpz2zscpqc']
[2024-01-31 14:58:49,623] {standard_task_runner.py:80} INFO - Job 168: Subtask aggregate_dim_table
[2024-01-31 14:58:49,659] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.aggregate_dim_table manual__2024-01-31T14:58:24.526071+00:00 [running]> on host d99f9e65d396
[2024-01-31 14:58:49,704] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=aggregate_dim_table
AIRFLOW_CTX_EXECUTION_DATE=2024-01-31T14:58:24.526071+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-31T14:58:24.526071+00:00
[2024-01-31 14:58:52,530] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-01-31 14:58:55,372] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-01-31 14:58:58,358] {logging_mixin.py:115} INFO - Successfully install pymysql
[2024-01-31 14:59:01,234] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-01-31 14:59:01,305] {python.py:173} INFO - Done. Returned value was: None
[2024-01-31 14:59:01,316] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_postgresql, task_id=aggregate_dim_table, execution_date=20240131T145824, start_date=20240131T145849, end_date=20240131T145901
[2024-01-31 14:59:01,368] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-31 14:59:01,391] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
